pyaedt/application/Design.py

```diff
@@ -2333,7 +2342,9 @@ def save_project(self, project_file=None, overwrite=True, refresh_obj_ids_after_
         """
         msg_text = "Saving {0} Project".format(self.project_name)
         self._messenger.add_info_message(msg_text, level='Global')
-        if project_file:
+        if project_file and not os.path.exists(os.path.dirname(project_file)):
+            os.makedirs(project_file)
```

Comment:
```suggestion
            os.makedirs(os.path.dirname(project_file))
```
It will fail otherwise.